test/Constraints/rdar35702810.swift

```diff
@@ -0,0 +1,21 @@
+// RUN: %target-swift-frontend -emit-sil -verify %s | %FileCheck %s
```

Comment:
Please put this test case in test/SILGen/function_conversions.swift together with the others, and add executable test to test/Interpreter/FunctionConversion.swift.

Reply:
Thanks! Will do!